app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,4 @@
 // Specific CSS for your home-page
+.center {
```

Comment:
bootstrap class `.text-center` does this precisely :)

Reply:
yes, I will delete this partial in the next push